### DISCUSSION TITLE
What are the benefits of enabling "Electron download mirror proxy" when scaffolding a new Electron project?

### DISCUSSION THREAD

**Discussion by adnanlah** [alex8088/electron-vite#405] [General]
Hello,

I understand that enabling "Electron download mirror proxy" will add a new `.npmrc` file with electron mirror links and it adds a electron download mirror in the `electron-builder.yml`.

I would like to know what are the benefits of this feature? and when do I need it.

Thanks.

**Comment by alex8088** [comment]
This option is designed for Chinese developers to solve network instability issues. If there are no installation issues with your, please ignore this.

**Reply by adnanlah** [reply]
Thank you very much for your huge work.

**Reply by MS90D3V** [reply]
Is it possible to document this somewhere?
When searching https://electron-vite.org/guide/

![Screenshot 2025-03-18 at 14 01 26](https://github.com/user-attachments/assets/554fef49-8771-433b-b0d5-ab019cef58a5)
